src/Domain/Fixture/FixtureCreator.php

```diff
@@ -319,6 +319,34 @@ private function addAllowedComposerPlugins(): void {
     $this->composerJsonHelper->addAllowedComposerPlugins($allowedComposerPlugins);
   }
 
+  /**
+   * Add packages defined in "allow-plugins" config of SUT to root composer.
+   */
+  private function addSutAllowedPluginsToRootComposer(): void {
+
+    $this->output->section("Adding Allowed Plugins from SUT");
+
+    $package = $this->options->getSut();
+    if ($package === NULL) {
+      $this->output->writeln("No SUT defined.");
+      return;
+    }
+
+    // Get plugins configured in "allow-plugins" config of SUT.
+    $allowedComposerPlugins = $this->subextensionManager
+      ->findAllowPluginsByPackage($package);
+    // Add plugins to the root composer fo fixture.
+    if (empty($allowedComposerPlugins)) {
```

Comment:
Oops, how did this get past me? Drupal coding standards require snake_case variables.

Also, how did it get past PHPCS? @secretsayan, would you please take a look at our PHPCS configuration and make sure it's working like we think it is?